pillar/journal-public.sls

```diff
@@ -3,20 +3,3 @@ journal:
         '30274': https://repro.elifesciences.org/example.html
     submit_url: https://submit.elifesciences.org/
     feature_xpub: false
-    calls_to_action:
```

Comment:
Should this be removed entirely, or does it need to exist, possibly with a minimal set of conceptually null data?

Reply:
This is the default for all environments, so needs to be left in as `[]` (empty list)